cli/azd/pkg/deploymentservice/deployments.go

```diff
@@ -0,0 +1,426 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package deploymentservice
```

Comment:
I'm wondering if `deployments` is a better package name here.  The `service` suffix may not be pulling it's weight, especially when we have our line limits in play?

Having the type name be `deploymentservice.DeploymentService` feels very long.  I wonder if `deployments.Service` is nicer

Reply:
I proposed a new name `azureapis` for the package, what do you think?

We can use that package for all the apis (just another way to reference the service, I think service is over-used already within azd).

This would give us:
- azureapis.Deployments
- azureapis.DeploymentOperations
- azureapis.ResourceManager
- azureapis.KeyVault
- etc.

We would save ourselves from having another meaning for a `service`.

Reply:
This feels like a pretty good tradeoff.  I'm not sure that's there's a ton of value in breaking the existing `AzCli` type up across packages, so having everything in `azureapis` makes sense.  This does let us break the existing large `AzCli` type into smaller pieces which is nice.

I wonder if `azapi` would be a better package name.  This sort of follows the convention of the existing Azure SDK packages which us use the `az` prefix.  It's also a little shorter which is nice.

Reply:
I like `azapi`.
will update